set.mm

```diff
@@ -22705,11 +22655,31 @@ derived from that of uniqueness ( ~ df-mo ).  (Contributed by Wolf
   $}
 
   ${
-    $d x y $.  $d y ph $.  $d x ps $.
+    $d x y $.  $d y z ph $.  $d x z ps $.
     mo4.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
     $( At-most-one quantifier expressed using implicit substitution.
-       (Contributed by NM, 26-Jul-1995.) $)
+       (Contributed by NM, 26-Jul-1995.)  Reduce axiom usage.  (Revised by Wolf
+       Lammen, 18-Oct-2023.) $)
     mo4 $p |- ( E* x ph <-> A. x A. y ( ( ph /\ ps ) -> x = y ) ) $=
+      ( vz wmo wa weq wi wal wex df-mo equequ1 pm2.27 alimdv imp expcom sylibr
+      com12 imbi12d cbvalvw adantr adantl anim12d equtr2 syl6 expd com3l syl5bi
+      pm2.43i exlimiv sylbi cbvexvw biimpri ax6evr pm3.2 imim1d ax7 com4r com23
+      syl8 eximdv mpi aleximi ax5e syl56 syl6ibr moabs exbii 3bitr4i impbii ) A
+      CGZABHZCDIZJZDKZCKZVMACFIZJZCKZFLZVRACFMZWAVRFWAVRWABDFIZJZDKZWAVRVTWECDV
+      OABVSWDECDFNUAUBZWFWAVRWFVTVQCVTWFVQVTWEVPDVNVTWEVOVNVTWEVOVNVTWEHVSWDHVO
+      VNVTVSWEWDAVTVSJBAVSOUCBWEWDJABWDOUDUECDFUFUGUHUIPTPTUJUKULUMVRBDGZVMVRBD
+      LZWHJWHVRWIWFFLZWHWIACLZVRWJCLWJWKWIABCDEUNUOVQAWJCAVQWJAVQHZVSFLWJFCUPWL
+      VSWFFAVQVSWFJAVSVQWFVSAVQWFJVSAHVPWEDVSAVPWEJAVPBVSWDAVPBVOVSWDJABVNVOABU
+      QURCDFUSVBUTQPRVAQVCVDRVEWJCVFVGBDFMZVHBDVISWBWJVMWHWAWFFWGVJWCWMVKSVL $.
+  $}
+
+  ${
+    $d x y $.  $d y ph $.  $d x ps $.
+    mo4OLD.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Obsolete version of ~ mo4 as of 18-Oct-2023.  (Contributed by NM,
+       26-Jul-1995.)  (New usage is discouraged.)
+       (Proof modification is discouraged.) $)
+    mo4OLD $p |- ( E* x ph <-> A. x A. y ( ( ph /\ ps ) -> x = y ) ) $=
```

Comment:
Since the proof is extremely shorter, I would like to keep this theorem as ~mo4ALT theorem.

Reply:
I'm not sure it's necessary to have an ALT proof here since that proof adds no insight (that is, proving ~xxx from ~xxxf using ~nfv is obvious and common: it should be kept only for the very first cases).

@wlammen : I would move your new ~mo4 before ~mo4f.

Reply:
I hinted in the comment to mo4f, and moved mo4 before mo4f